### DISCUSSION TITLE
link play

### DISCUSSION THREAD

**Discussion by enchanteddaemon** [Lost-MSth/Arcaea-server#88] [Q&A]
what if i delete or not add linkplay feature

**Comment by Lost-MSth** [comment]
You can close it via setting `SET_LINKPLAY_SERVER_AS_SUB_PROCESS = False`. It will not influence the main feature.

**Comment by enchanteddaemon** [comment]
> You can close it via setting `SET_LINKPLAY_SERVER_AS_SUB_PROCESS = False`. It will not influence the main feature.

where is that

**Reply by Lost-MSth** [reply]
In `config.py`

**Reply by enchanteddaemon** [reply]
oh okay

**Reply by enchanteddaemon** [reply]
do you have discord so i can chat with you

**Reply by Lost-MSth** [reply]
For personal contact, email please. It is too hard for me to use discord or some other social network.
